internal/packages/packages.go

```diff
@@ -5,94 +5,63 @@
 package packages
 
 import (
-	"encoding/json"
-	"io/ioutil"
 	"os"
 	"path/filepath"
 
+	"github.com/elastic/go-ucfg"
+	"github.com/elastic/go-ucfg/yaml"
 	"github.com/pkg/errors"
-	"gopkg.in/yaml.v3"
 )
 
 const (
 	// PackageManifestFile is the name of the package's main manifest file.
 	PackageManifestFile = "manifest.yml"
 
-	// DataStreamManifestFile is the name of the dataStream's manifest file.
+	// DataStreamManifestFile is the name of the data stream's manifest file.
 	DataStreamManifestFile = "manifest.yml"
 )
 
-// VarValue represents a variable value as defined in a package or dataStream
-// manifest file.
-type VarValue struct {
-	scalar string
-	list   []string
+// Variable is an instance of configuration variable (named, typed).
+type Variable struct {
+	Name    string      `config:"name" json:"name" yaml:"name"`
+	Type    string      `config:"type" json:"type" yaml:"type"`
+	Default interface{} `config:"default" json:"default" yaml:"default"`
```

Comment:
Now that we're introducing the `config` struct tags for `ucfg`'s benefit, do we still need to keep the `yaml` struct tags?

Reply:
I think you're right, although I have to check whether we don't call `yaml.Marshal` somewhere...